packages/editor/src/components/post-fields/index.js

```diff
@@ -25,19 +25,24 @@ import {
 } from '@wordpress/icons';
 import { __experimentalHStack as HStack, Icon } from '@wordpress/components';
 import { useSelect } from '@wordpress/data';
-import { useEntityRecords, store as coreStore } from '@wordpress/core-data';
+import {
+	useEntityRecord,
+	useEntityRecords,
+	store as coreStore,
+} from '@wordpress/core-data';
+import { privateApis as routerPrivateApis } from '@wordpress/router';
 
 /**
  * Internal dependencies
  */
-import {
-	LAYOUT_GRID,
-	LAYOUT_TABLE,
-	LAYOUT_LIST,
-	OPERATOR_IS_ANY,
-} from '../../utils/constants';
-import { default as Link, useLink } from '../routes/link';
-import Media from '../media';
+import { unlock } from '../../lock-unlock';
```

Comment:
This is the gist of this PR. To be able to move post-fields from edit-site to the editor package, we need to remove any edit-site dependency:

- DataViews-related constants: make them inline.
- The `Media` component: make it inline, as it was only used by post-fields.
- Link, useLink: move to `@wordpress/router` package.